charts/fluentd/templates/logger-fluentd-daemon.yaml

```diff
@@ -33,6 +33,15 @@ spec:
             memory: {{.Values.limits_memory}}
 {{- end}}
 {{- end}}
+{{- if and (.Values.syslog_host) (.Values.syslog_port)}}
```

Comment:
they should now be `.Values.syslog.host` and `.Values.syslog.port` as of the most recent change